product-scenarios/4-gateway/4.1-Extend-the-reach-for-existing-or-legacy-applications/4.1.1-Expose-back-ends-exposed-over-HTTP-as-SOAP-service/4.1.1.1-Expose-a-SOAP-service-as-SOAP-service-using-proxy-service/src/test/resources/testng.xml

```diff
@@ -17,6 +17,9 @@
   -->
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 <suite name="ESBTestSuite" parallel="false">
+    <listeners>
+        <listener class-name="org.wso2.carbon.esb.scenario.test.common.testng.listeners.TestExecutionListener"/>
```

Comment:
Shall we add a comment stating what this listener does?

Reply:
+1, added